play/PlayerRules/CharacterCreation/06_Skills.txt

```diff
@@ -1,112 +1,119 @@
-//See the License for the specific language governing permissions and
-//limitations under the License.
+/*=============================================================================
+ * Skills List
+ * This document lists all the skills used in Compound_X. Skills augment your
+ * natural ability. Anyone who is strong enough can kick in a door, but if you
+ * are specially trained, you can much more effectively accomplish the task.
+ *===========================================================================*/
+//Dex
+Acrobatics: Rolling and tumbling come naturally to you. Useful for dodging, 
+jumping, balancing, and other fine body-control.
+//Per
+Awareness: your character''s passive perception skill including all 6 senses. This differs from being able to look intently for something particular (Search).
+//Cha
+Deceive: You use your silver tongue to talk your way through situations with 
+ease.
+//Str
+Door Kicking: You''ve been trained in the subtle art of viscously and 
+unceremoniously removing a door from it''s frame for rapid entry.
+//Int
+Hacking: Used to gain outside access to computer based systems.
+//Dex
+Hand to Hand: See damage rules under Weapons listing. Can be used to attack 
+any opponent that has a standard health bar.
+//Dex
+Hand to Hand [Expert]: Experts in hand to Hand can perform feats unimaginable 
+to mere mortals. They can perform silent take downs, disarmaments, and can 
+fight against  edged weapons without a penalty. However, they have to have 
+specifically mentioned expert training in their background. Standard mechanics,
+or scientists, or farmers will not be able to use this skill. A Buddhist 
+Fighting Monk on the other hand, would get this skill for free. Talk to your 
+GM. For the most part, anyone can take H-H Expert as a skill, but they must 
+take the Melee Combat Expert Feat.
+///Int
+Knowledge [X]: You may have gone to college to get a degree in X, or maybe you 
+heard all about X from your local reputable information source. Either way, 
+you know a lot about X. Options:
+	Local
+	Criminal
+	Nanites
+	Sorcery
+	Technology
+	Alien
+	Military
+//Int
+Medicine: If health is over 50, DC 30. Any amount over 30 is restored to target
+''s health. Rolls under 30 deal d10 bleed damage. If health is under 50, DC 50
+. Any amount over 50 is restored to target''s health. Rolls under 30 deal d10 
+bleed damage. Medicine skill checks can be performed even if untrained in 
+medicine, but the user takes a -10%. Medicine packs decrease the DC by 10, 
+increase the amount healed by 30, and negate bleed damage on a failed roll.
+//Int
+Navigate: you can read a map, congratulations. Any time you are lost, or need 
+to decide where to go, this skill will channel your inner survivorman and 
+enable you to have a better idea of where you should face.
+//Dex
+Piloting: You have been trained in the use and operation of many vehicles, and 
+even if you donâ€™t recognize what it is, you can probably figure out how to use 

```

Comment:
UTF-8 encoding versus ASCII I believe. we should fix it to the lowest common denominator.